src/main/groovy/com/marklogic/gradle/task/roxy/scaffoldTemplates/SampleTestFiles.groovy

```diff
@@ -0,0 +1,158 @@
+package com.marklogic.gradle.task.roxy
+
+class SampleTestFiles {
+
+    static String getSampleTestsXqy() {
+        return """
+xquery version '1.0-ml';
+
+import module namespace c = 'http://marklogic.com/roxy/test-config' at '/test/test-config.xqy';
+import module namespace test = 'http://marklogic.com/roxy/test-helper' at '/test/test-helper.xqy';
+import module namespace sec= 'http://marklogic.com/xdmp/security' at  '/MarkLogic/security.xqy';
+import module namespace cfg = 'http://thieme.com/synone/config' at '/app/config/config.xqy';
+
+
+declare option xdmp:mapping 'false';
+
+(: using global variable from app/config/config.xqy :)
+
+declare variable \\\$admin-user as xs:string := \\\$cfg:SEARCH-USER;
```

Comment:
I'd rather not have references to config.xqy because that assumes that the Roxy config.xqy module is being used, which I don't think is likely on an ml-gradle project. Do these variables even need to be declared?